zone.go

```diff
@@ -97,6 +100,19 @@ func (z *Zone) WhoisURL() string {
 	return ""
 }
 
+// RDAPURLs returns the set of RDAP URL endpoints for a subdomain
+// of the zone. It first searches the specific zone, then the parent,
+// returning nil if none found.
+func (z *Zone) RDAPURLs() []string {
+	if len(z.rdapURLs) != 0 {
+		return z.rdapURLs
+	}
+	if z.Parent != nil {
+		return z.Parent.RDAPURLs()
+	}
```

Comment:
This could be interesting for private zones like `us.com`, which would return the `com` RDAP URLs, which are unlikely to provide a useful or authoritative result.

Reply:
Remove the Parent call and just return value of struct as-is? Also, keep struct field as unexported I assume.

Reply:
yeah